### PR TITLE
Set the minimal runtime version to NET 6

### PR DESCRIPTION
**Pull request by JoeRobich**

### CODE REVIEW

**Review by jmarolf — APPROVED** [review]
I assume this needs to be merged before we merge https://github.com/dotnet/format/pull/1276

**Comment by JoeRobich** [comment]
> I assume this needs to be merged before we merge #1276

I think we are good to merge both.

File: tests/dotnet-format.UnitTests.csproj [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
 
     <!-- Always run on the latest runtime installed. -->
-    <RuntimeFrameworkVersion>2.1.0-rc1</RuntimeFrameworkVersion>
+    <RuntimeFrameworkVersion>6.0.0-preview.2.21154.6</RuntimeFrameworkVersion>
```
Comment (JoeRobich) [inline tests/dotnet-format.UnitTests.csproj:13]:
```suggestion
    <RuntimeFrameworkVersion>6.0.0-preview.3.21122.7</RuntimeFrameworkVersion>
```

File: src/dotnet-format.csproj [hunk 2]
```diff
@@ -19,7 +19,7 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
 
     <!-- Always run on the latest runtime installed. -->
-    <RuntimeFrameworkVersion>3.1.0-preview3.19553.2</RuntimeFrameworkVersion>
+    <RuntimeFrameworkVersion>6.0.0-preview.2.21154.6</RuntimeFrameworkVersion>
```
Comment (JoeRobich) [inline src/dotnet-format.csproj:22]:
```suggestion
    <RuntimeFrameworkVersion>6.0.0-preview.3.21122.7</RuntimeFrameworkVersion>
```

**Pull request merged**